### DISCUSSION TITLE
abort sync, triggerSource=manual, error while getting_remote-Extra_meta

### DISCUSSION THREAD

**Discussion by ddfun** [remotely-save/remotely-save#105] [General]
#77 
help!!
remotely-save-1651506378313:
abort sync, triggerSource=manual, error while getting_remote-Extra_meta

on my Samsung Note 20 Ultra Android 12
Ver. 0.3.22
![159881](https://user-images.githubusercontent.com/104687527/166265479-87f60029-250e-4546-85ad-ddb383fe2c09.jpg)

**Comment by fyears** [comment]
your issue is different from that in #77 

what remote service are you using?

**Comment by fyears** [comment]
It should be a dup of https://github.com/remotely-save/remotely-save/issues/104  (I am assuming you are using onedrive)

Please update the plugin to https://github.com/remotely-save/remotely-save/releases/tag/0.3.23 and try again. Thanks!

**Reply by ddfun** [reply]
Thanks , it's working now.
